db/seeds/playbooks/wronged.seeds.rb

```diff
@@ -0,0 +1,299 @@
+       playbook_id: @wronged.id,
+       description: 'With your signature weapon (see
+your gear, below), you get +1 to *kick some ass*.'
+   }].each do |move|
+    Move.find_or_create_by!(move)
+  end
+
+  #####
+  # Gear
+  #####
+
+  # Note: Signature weapon could also be:
+  # Specialist weapons for destroying your foes (e.g.
+  # wooden stakes and mallet for vampires, silver dagger
+  # for werewolves, etc.).
+  # 4-harm against the specific
+  # creatures it targets, 1-harm otherwise, and other
+  # tags by agreement with the Keeper
+
+  [
+    {
+      name: 'Sawn-off shotgun',
+      harm: 3,
+      tag_list: %w[hand/close messy loud reload]
+    },
+    {
+      name: 'Hand cannon',
+      harm: 3,
+      tag_list: %w[close loud]
+    },
+    {
+      name: 'Fighting knife',
+      harm: 2,
+      tag_list: %w[hand quiet]
+    },
+    {
+      name: 'Huge sword or huge axe',
+      harm: 3,
+      tag_list: %w[harm hand messy heavy],
+    },
+    {
+      name: 'Enchanted dagger',
+      harm: 2,
+      tag_list: %w[harm hand magic]
+    },
+    {
+      name: 'Chainsaw',
+      harm: 2,
+      tag_list: %w[hand messy unreliable loud heavy]
+    },
+    {
+      name: '.38 resolver',
+      harm: 2,
+      tag_list: %w[close reload loud]
+    },
+    {
+      name: '9mm',
+      harm: 2,
+      tag_list: %w[close loud]
+    },
+    {
+      name: 'Hunting rifle',
+      harm: 2,
+      tag_list: %w[far loud]
+    },
+    {
+      name: 'Shotgun',
+      harm: 3,
+      tag_list: %w[close messy loud]
+    },
+    {
+      name: 'Big knife',
+      harm: 1,
+      tag_list: %w[hand]
+    },
+    {
+      name: 'Brass knuckles',
+      harm: 1,
+      tag_list: %w[close hand stealthy]
+    },
+    {
+      name: 'Assault rifle',
+      harm: 3,
+      tag_list: %w[close area loud reload]
+    },
+  ].each do |gear_attrs|
+    gear = Gear.find_or_create_by!(
+        name: gear_attrs[:name],
+        playbook: @wronged
+    )
+    gear.update!(gear_attrs)
+  end
+
+  #####
+  # Improvements
+  #####
+  [
+    {
+      description: 'Get +1 Tough, max +3',
+      type: 'Improvements::RatingBoost',
+      stat_limit: 3,
+      rating: :tough,
+      playbook: @wronged
+    },
+    {
+      description: 'Get +1 Cool, max +2',
+      type: 'Improvements::RatingBoost',
+      stat_limit: 2,
+      rating: :cool,
+      playbook: @wronged
+    },
+    {
+      description: 'Get +1 Sharp, max +2',
+      type: 'Improvements::RatingBoost',
+      stat_limit: 2,
+      rating: :sharp,
+      playbook: @wronged
+    },
+    {
+      description: 'Get +1 Weird, max +2',
+      type: 'Improvements::RatingBoost',
+      stat_limit: 2,
+      rating: :weird,
+      playbook: @wronged
+    },
+    {
+      description: 'Take another Wronged move',
+      type: 'Improvements::AnotherMove',
+      playbook: @wronged,
+      stat_limit: 0
+    },
+    {
+      description: 'Take another Wronged move',
+      type: 'Improvements::AnotherMove',
```

Comment:
I believe this should actually be a Improvements::PlaybookMove because it belongs to your playbook, not another playbook.